src/mpyl/reporting/formatting/markdown.py

```diff
@@ -16,14 +18,24 @@ def summary_to_markdown(summary: TestRunSummary):
            f"💔 {summary.errors} 🙈 {summary.skipped}"
 
 
+def __add_link(name: str, output: Output) -> str:
```

Comment:
maybe this is nitpicking but it confused me while reading the function: it's called `add_link` but it actually returns a link (or name)?

Reply:
Not sure I understand the confusion 🤔 . How would you rather call this function?

Reply:
eh it's not a big deal anyway